jsonnet/components/node-exporter.libsonnet

```diff
@@ -147,18 +147,11 @@ function(params)
                       // gather that data (especially for bare metal clusters), and
                       // add flags to collect the node_cpu_info metric + metrics
                       // from the text file.
-                      args: [a for a in c.args if (a != '--no-collector.hwmon') && !std.startsWith(a, '--collector.netclass.ignored-devices')] +
+                      args: [a for a in c.args if !std.member(['--no-collector.hwmon', '--collector.cpufreq', '--no-collector.cpufreq'], a) && !std.startsWith(a, '--collector.netclass.ignored-devices')] +
```

Comment:
IIUC this change has no effect on the generated assets. Is this to prevent any future breakage if upstream jsonnet enables/disables cpufreq?

Reply:
yes, this prevents future upstream updating arguments for `cpufreq` collector and changes our default settings.

Reply:
shall we remove this filter, too?

Reply:
yes I'd remove to keep the code cleaner. If upstream decides to add a cpufreq flag in the future, we'll know for sure from the failing e2e tests and we'll fix it then.